pontoon/base/templatetags/helpers.py

```diff
@@ -263,36 +263,43 @@ def dict_html_attrs(dict_obj):
     ))
 
 
-def _serialize_elements(elements):
-    """Serialize text elements and placeables into a simple string."""
+def _get_default_variant(variants):
+    """Return default variant from the list of variants."""
+    return filter(lambda x: x.default, variants)[0]
+
+
+def _serialize_value(value):
+    """Serialize AST values into a simple string."""
     response = ''
 
-    for element in elements:
-        if isinstance(element, ast.TextElement):
-            response += element.value
+    if hasattr(value, 'variants'):
```

Comment:
Please use `isinstance` here and later. I don't think duck typing is the right approach for working with the AST. The logic is specific to a data type, which happens to have fields named this way.

```python
if isinstance(value, ast.VariantList):
```